src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerOfAPerson = (
+    sex === 'm'
+      ? 'wife'
+      : 'husband'
+  );
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+      <p className="Person__partner">
+        {
```

Comment:
You might want to edit these lines to make them easier to read.

Reply:
done bro

Reply:
Did you push and deploy? Because I can't see it yet.

Reply:
yeah, I don't see it either